js/functions.js

```diff
@@ -0,0 +1,30 @@
+// Функция для проверки длины строки
+
+let lessOrEqual = (string, length) => string.length <= length;
+console.log(lessOrEqual("бубубу", 12));
```

Comment:
Обсудим завтра, чтобы линтер не ругался, а пока так
```suggestion
lessOrEqual("бубубу", 12)
```